src/lib/Scenes/Artwork/Components/CommercialButtons.tsx

```diff
@@ -0,0 +1,226 @@
+import { Button, Spacer } from "@artsy/palette"
+import { CommercialButtons_artwork } from "__generated__/CommercialButtons_artwork.graphql"
+import { CommercialButtonsOfferOrderMutation } from "__generated__/CommercialButtonsOfferOrderMutation.graphql"
+import { CommercialButtonsOrderMutation } from "__generated__/CommercialButtonsOrderMutation.graphql"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import React from "react"
+import { commitMutation, createFragmentContainer, graphql, RelayProp } from "react-relay"
+
+interface Props {
+  artwork: CommercialButtons_artwork
+  relay: RelayProp
+  editionSetID?: string
+}
+
+export interface State {
+  isCommittingCreateOrderMutation: boolean
+  isCommittingCreateOfferOrderMutation: boolean
+}
+
+export class CommercialButtons extends React.Component<Props, State> {
+  state = {
+    isCommittingCreateOfferOrderMutation: false,
+    isCommittingCreateOrderMutation: false,
+  }
+
+  onMutationError(error) {
+    // FIXME: Handle error
+    console.error("src/lib/Scenes/Artwork/Components/CommercialButtons.tsx", error)
+  }
+
+  handleCreateOrder() {
```

Comment:
When we build out the `Buy now` button for auctions, we may want to re-use this logic.